includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -2276,4 +2338,161 @@ protected function guess_modal_role( DOMElement $modal ) {
 		// None of the roles we are looking for match any of the classes.
 		return Role::DIALOG;
 	}
+
+	/**
+	 * Evaluates the given XPath expression and give first Element if exist.
+	 *
+	 * @param string  $expression   The XPath expression to execute.
+	 * @param Element $context_node The optional context node can be specified for doing relative XPath queries.
+	 *                              By default, the queries are relative to the root element.
+	 *
+	 * @return Element|bool Return Element if exists otherwise False.
+	 */
+	protected function get_first_element( $expression, $context_node = null ) {
+
+		if ( empty( $expression ) ) {
+			return false;
+		}
```

Comment:
Is there ever a scenario where an empty expression is passed? If so, this should probably be raised as a warning so it can be caught.

```suggestion
```